cime_config/testdefs/testlist_clm.xml

```diff
@@ -3065,15 +3065,65 @@
       <option name="comment">Transient production test with clm6, t232, and Crujra while the defaults have not changed</option>
     </options>
   </test>
-  <test name="SMS_D_Ld5" grid="f09_g17" compset="ISSP126Clm50BgcCrop" testmods="clm/datm_ssp126_anom_forc">
```

Comment:
FYI: In #3125 I removed this test from the testlist, since it's broken there. So when you merge that into here, this will come up as a conflict. But, you'll just add this version back in at this point. So it's an easy conflict to take care of.